view/frontend/templates/instant/stats.phtml

```diff
@@ -2,7 +2,7 @@
     {{#hasOneResult}}<strong>1</strong> <?php echo $block->escapeHtml(__('result')); ?> found{{/hasOneResult}}
 
     {{#hasManyResults}}
-        {{^hasNoResults}}{{first}}-{{last}} out of{{/hasNoResults}}
+        {{^hasNoResults}}{{first}}-{{last}} <?= __("out of")?>{{/hasNoResults}}
```

Comment:
Hello, could you please change `<?= __("out of")?>`  to` <?php echo $block->escapeHtml(__('out of')); ?>` cause this syntax isn't allowed by our quality tools check.
Thanks in advance.

Reply:
Thanks for the pointer! I've resolved it with my last commits.